src/callback_requests.rs

```diff
@@ -34,6 +34,10 @@ pub enum CallbackRequestType {
         /// String pointing to the object (e.g.: `registry.testing.lan/busybox:1.0.0`)
         image: String,
     },
+    OciManifest {
```

Comment:
I would like to also have a comment explaning what does this callback request entail, like the others.